portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+let i = 0;
+let facts = [
+  "Digital Arts and Sciences Engineer ",
+  "Full-Stack Developer ",
+  "Scrum Master ",
+  "Google SPS participant! >:) "
+];
+let txt = "I'm a " + facts[0];
+let speed = 50;
+let backspace = false;
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+window.onload = function() {
+  typeWriter();
+  i = 0;
+  txt = "I'm a " + facts[0];
+  backspace = false;
+};
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function typeButton() {
+  typeWriter();
+  i = 0;
+  txt = "I'm a " + facts[Math.floor(Math.random() * facts.length)];
+  backspace = false;
+}
+
+function typeWriter() {
+  if (i <= txt.length && !backspace) {
+    document.getElementById("typed").innerHTML = txt.substring(0, i) + "|";
+    i++;
+    if (i === txt.length + 1) {
+      backspace = true;
+      setTimeout(wait(), 15000);
```

Comment:
I notice that below you call `setTimeout(typeWriter, ...)` but here you call `setTimeout(wait(), ...)`. I think maybe you meant to call `setTimeout(wait, ...)` here without the `()` after `wait`? That way you're passing the `wait` function in as a parameter, rather than calling the `wait()` function. Does that make sense?